.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,23 @@
+name: E2E tests
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      E2E_TESTS: true
+      E2E_LOGS: true
+      CI_VERBOSE: true
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Run tests
+        run: make test-e2e
+      # - name: Archive test logs
+      #   if: always()
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: e2e-logs
+      #     path: e2e-logs-*/
```

Comment:
Just checking, we are going to uncomment this once [Add logging into e2e framework](https://polygon.atlassian.net/jira/software/c/projects/EVM/boards/5?modal=detail&selectedIssue=EVM-51) gets implemented?

Reply:
Yup see https://github.com/0xPolygon/polygon-edge/pull/771

Reply:
Ok great. In that case, I'd consider maybe even removing this part in order to avoid unnecessary merge conflict (although it is pretty straight forward resolving it). WDYT I'm ok with either approach.

Reply:
Let's go with it as the next PR is branched from this branch so, no conflict